### PR TITLE
Event ticket Creation

### PR DESCRIPTION
**Pull request by marinPh**
A pull request to add tickets from emails to users and ticket tracking system with tests
this is the simple page to put the email, and add the ticket
Screenshot 2024-05-13 185653

here are both both possible replies

after success, we add the ticket to the global tickets and the user tickets
Screenshot 2024-05-13 185743
Screenshot 2024-05-13 185816

### CODE REVIEW

File: scrapping/test.ipynb [hunk 1]
```diff
@@ -292,6 +292,13 @@
     "    \n",
     "\n"
    ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": []
```
Comment (marinPh) [inline scrapping/test.ipynb:301]: I don't know why the jupyter notebook used to scrap added itself a cell and how I pushed it. However it does not affect anything

File: app/src/androidTest/java/com/swent/assos/CreateTicketTest.kt [hunk 2]
```diff
@@ -53,4 +90,55 @@ class CreateTicketTest : SuperTest() {
...
+        }
+        step("click") { composeTestRule.onNodeWithText("Submit").performClick() }
+
+        composeTestRule.waitForIdle()
+        // wait until test "Ok" is displayed
+        composeTestRule.waitUntil(
+            condition = { composeTestRule.onNodeWithText("Ok").isDisplayed() },
+            timeoutMillis = 5000)
+
+        step("dismiss") { composeTestRule.onNodeWithText("Ok").performClick() }
```
Comment (alicepotter) [inline app/src/androidTest/java/com/swent/assos/CreateTicketTest.kt:113]: I don't get how you can differentiate the success dialog with the failure dialog if you search for a node with the "Ok" text as both dialogs have that same text.

**Review by alicepotter — APPROVED** [review]
Great job ! I recommend you change the name of the PR

File: app/src/androidTest/java/com/swent/assos/CreateTicketTest.kt (hunk 2 above)
Comment (marinPh) [reply app/src/androidTest/java/com/swent/assos/CreateTicketTest.kt:113]: that is why I check at the end the number of tickets in success

**Pull request merged**